docs/contributing-to-airbyte/submit-new-connector.md

```diff
@@ -1,42 +1,25 @@
 
-1. Verify if there is an existing [Issue](https://github.com/airbytehq/airbyte/issues?q=is%3Aopen+is%3Aissue+label%3Aarea%2Fconnectors+-label%3Aneeds-triage+label%3Acommunity)
-2. If you don't find an existing issue, [Request a New Connector](https://github.com/airbytehq/airbyte/issues/new?assignees=&labels=area%2Fconnectors%2Cnew-connector&projects=&template=5-feature-new-connector.yaml)
+1. Verify if there is an existing [Discussion](https://github.com/airbytehq/airbyte/discussions/categories/new-connector-request) for a connector you have in mind
+2. If you don't find an existing issue, [Request a New Connector](https://github.com/airbytehq/airbyte/discussions/new?category=new-connector-request)
 
 This will enable our team to make sure your contribution does not overlap with existing works and will comply with the design orientation we are currently heading the product toward. If you do not receive an update on the issue from our team, please ping us on [Slack](https://slack.airbyte.io)!
 
-#### Code your contribution
+#### Build the connector
 
-1. To contribute to a connector, fork the [Connector repository](https://github.com/airbytehq/airbyte).
-2. Open a branch for your work
-3. Code the change
-4. Ensure all tests pass. For connectors, this includes acceptance tests as well.
-5. Update documentation in `docs/integrations/<connector-name>.md`
-6. Make sure your contribution passes our [QA checks](./resources/qa-checks.md)
+See [the Connector Development](../connector-development/README.md) guide for more details on how to build a connector. For most API source connectors, Connector Builder is the way to go.
 
 #### Open a pull request
 
-1. Rebase master with your branch before submitting a pull request.
-2. Open the pull request.
-3. Follow the [title convention](./resources/pull-requests-handbook.md#pull-request-title-convention) for Pull Requests
-4. Link to an existing Issue
-5. Update the [description](./resources/pull-requests-handbook.md#descriptions)
-6. Wait for a review from a community maintainer or our team.
-
-#### 4. Review process
-
-When we review, we look at:
-
-- ‌Does the PR add all existing streams, pagination and incremental syncs?
-- Is the proposed solution reasonable?
-- Is it tested? \(unit tests or integation tests\)
-  ‌Once your PR passes, we will merge it 🎉.
+1. Make sure your connector passes `airbyte-ci connectors test` tests. [Here's a guide on how to run them](../connector-development/testing-connectors/README.md).
+2. Make sure you include the README, documentation, and an icon for your connector. Without them, one of the CI checks will fail.
+3. Follow the [pull request convention](./resources/pull-requests-handbook.md#pull-request-title-convention)
+4. Wait for a review from a community maintainer or our team. We generally look for:
+  - Does this PR cover authentication, pagination, incremental syncs where applicable?
+  - ‌Does the PR add reasonable list of streams?
```

Comment:
I think this might need some clarification.

Reply:
Yep, but that's way too dynamic to point to an existing doc _right away_, and will change once we ship Builder contrib flow.